### PR TITLE
INFRA-1175: allow setting timestamp column name through config

### PR DESCRIPTION
**Pull request by nikhilpandit**
Quite a few of the event tables have timestamp in them, but the name of the timestamp column is not always the same. This allows the firehose config to specify a name for the timestamp column in Redshift.

### CODE REVIEW

**Comment by nikhilpandit** [comment]
cc @drhurd (since we spoke about this earlier today)

**Comment by nathanleiby** [comment]
Could update the unit tests to cover this

File: firehose_output.go [hunk 1]
```diff
@@ -40,8 +45,19 @@ func (f *FirehoseOutput) Run(or pipeline.OutputRunner, h pipeline.PluginHelper)
 			continue
 		}
 
+		if f.timestampColumn != "" {
+			// add timestamp to column named in timestampColumn
```
Comment (nathanleiby) [inline firehose_output.go]: add timestamp -> add Heka message's timestamp

**Comment by nathanleiby** [comment]
lgtm

**Comment by drhurd** [comment]
:+1:

File: firehose_output.go (hunk 1 above)
Comment (nikhilpandit) [inline firehose_output.go]: updated: https://github.com/Clever/heka-clever-plugins/commit/64a7aefeae1ae93ad6d75547b965ac5fb521da58

**Comment by nikhilpandit** [comment]
added a test to test output when `TimestampColumn` is specified.

**Pull request merged**